### PR TITLE
Fix: Several mistakes in docs

### PR DESCRIPTION
**Pull request by wizard7377**
Fixes for several trivial mistakes in docs, including spelling, grammar, and incorrect/outdated information.

### CODE REVIEW

**Review by Mishura4 — CHANGES_REQUESTED** [review]
This review only applies to `advanced_reference/coroutines.md`, I haven't looked at the other files.

`advanced_reference/coroutines.md` is going to receive a lot of changes in #763, please wait until it is merged (or remove the changes from this PR and I can add them on my end)

File: docpages/advanced_reference/automating-with-jenkins.md
Comment (Jaskowicz1) [inline docpages/advanced_reference/automating-with-jenkins.md:53]: No need to change `/var/lib/jenkins/` to `/var/lib/jenkins`

**Pull request closed without merging**
